Composer/plugins/azurePublish/src/index.ts

```diff
@@ -269,8 +258,8 @@ class AzurePublisher {
         subscriptionID,
         publishName,
         environment,
-        luisAuthoringKey,
-        luisAuthoringRegion,
+        luisAuthoringKey: provision?.luis?.authoringKey,
```

Comment:
It is not necessary to pull these in - they are already being pulled out of the appsettings.deployment.json file inside the deploy function in `botProjectDeploy`

```
      if (luisSettings) {
        // if luisAuthoringKey is not set, use the one from the luis settings
        luisAuthoringKey = luisAuthoringKey || luisSettings.authoringKey;
        luisAuthoringRegion = luisAuthoringRegion || luisSettings.region;
        luisEndpointKey = luisSettings.endpointKey;
      }
```